README.md

```diff
@@ -17,14 +17,38 @@ To deploy manually
 
 To deploy using serverless:
 - install serverless (available via npm and choco)
-- set up an appropriate IAM user in your AWS account and obtain a key and secret
+- set up an appropriate IAM user (see below) in your AWS account and obtain a key and secret
 - set up AWS credentials locally with 
 `serverless config credentials --provider aws --key AKIAIOSFODNN7EXAMPLE --secret wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY`
 See the guide at https://serverless.com/framework/docs/providers/aws/guide/credentials/ for more detail and alternative ways to do this
 - Run serverless deploy and note down the endpoint given
+- Logs can be found in AWS Lambda -> Monitoring
 
 To hook into Gitlab:
 - Go to Settings->Integrations in Gitlab
 - Add a new webhook triggered by `Comments` and `Merge request events`. Hook URL should be the endpoint you've noted down. Untick security options
 - Click Test->`Note events` or `Merge requests events`
-- Result should appear at the top of the page
+- Result should appear at the top of the page: ![Gitlab success message](./gitlab-success-message.png)
+- Add a `.clabot` file in the root of the project with a whitelist of the gitlab usernames that are cla-signed:
+```
+{
+  "contributors": ["<gitlab-username>"]
+}
+```
+- Get one of the whitelisted contributors to make a comment including the text "@gitlab-cla-bot check" on a merge request or create a merge request.
+
```

Comment:
I think it shouldn't matter who comments on the MR - it's the committers who are checked against the whitelist.

Reply:
Though I haven't checked this :)

Reply:
Would this text be more accurate?
```
- Create a merge request or make a comment including the text "@gitlab-cla-bot check" on a merge request and the bot should respond.
```

Reply:
👍